recipes/glib/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set major_minor = "2.48" %}
+{% set version = major_minor + ".0" %}
+{% set sha256 = "744be6931ca914c68af98dc38ff6b0cf8381d65e335060faddfbf04c17147c34" %}
```

Comment:
Yeah, we had a pretty long discussion in this PR ( https://github.com/conda-forge/staged-recipes/pull/177 ), which took jinja templates to the next level. See the 16 comment outdated diff (unfortunately GitHub is terrible at linking to outdated diffs). We decided that we should allow some freedoms to recipe maintainers and we came to the consensus that this was one of them.

Personally, I kind of liked this flourish as it allowed people to reuse an existing recipe from `staged-recipes` as a template and merely sub out the stuff at the top. Admittedly, I don't think building a GNOME package will be quite so cookie cutter as a Python package, but I'm still ok with allowing it if the maintainer is willing to do the upkeep this way.